src/beanahead/reconcile.py

```diff
@@ -362,7 +365,8 @@ def confirm_single(
     print(
         f"{utils.SEPARATOR_LINE}Expected Transaction:\n"
         f"{utils.compose_entries_content(x_txn)}\n"
-        f"Incoming Transaction:\n{utils.compose_entries_content(matches[0])}"
+        f"Incoming Transaction:\n{utils.compose_entries_content(matches[0])}",
+        file=sys.stderr,
```

Comment:
Could you clarify for me why the hook requires that the print goes to the stderr rather than stdout. Thanks.

Reply:
When running as part of the importer, stdout is typically redirected to the output file: `./foo.importer.py extract --download ./input ... >> foo.beancount`

Consequently, for any user interaction to take place, it would have to do so on stderr. It's a pragmatic solution to a problem that could otherwise only be addressed through an upstream refactoring.

Reply:
Of course 🤦‍♂️.
Thanks